templates/CMakeLists.txt.template

```diff
@@ -345,49 +370,53 @@
   #   Add custom commands to process ``.proto`` files to C++ using protoc and
   #   GRPC plugin::
   #
-  #     protobuf_generate_grpc_cpp [<ARGN>...]
+  #     protobuf_generate_grpc_cpp <FILE_LOCATION> <IMPORT_PATH>
   #
-  #   ``ARGN``
-  #     ``.proto`` files
+  #   ``FILE_LOCATION``
+  #     The relative path of the ``.proto`` file to the project root
+  #   ``IMPORT_PATH``
```

Comment:
nit: you're calling this arg "import_path" which would suggest that it's a directory name, but from the generated files I can see it's the full file name (not just the path).